### ISSUE TITLE
NetKAT Parser Doesn't Allow Output to Controller or Query

### ISSUE THREAD

**Issue by craig-riecke** [frenetic-lang/frenetic#493]
The NetKAT parser doesn't handle `Mod(Location(Pipe(n))` or `Mod(Location(Query(n))`  Unfortunately there's nothing in the academic papers that outlines the "official" syntax for it.   In the absence of a better suggestion, I could implement it as:

```
port := pipe(http)
port := query(ip_packets)
```

And then it kinda looks orthogonal to sending to physical ports, as in:

```
port := 1 | port := 2 | port := query(broadcasts)
```

**Comment by smolkaj** [comment]
Could we make `http` and `ip_packets` explicit strings?
```
port := pipe("http")
port := query("ip_packets")
```

**Comment by craig-riecke** [comment]
+1.  That makes it really clear that `http` and `ip_packets` are not part of the NetKAT syntax.

**Comment by craig-riecke** [comment]
Fixed by #496.
